modules/custom/core/govcms_security/src/GovcmsFileConstraintInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Drupal\govcms_security;
+
+/**
+ * Provides an interface for constrants on files uploaded.
+ */
+interface GovcmsFileConstraintInterface {
+
+  /**
+   * A list of blocked file extensions.
+   */
+  public const BLOCKED_EXTENSIONS = ['doc', 'xls', 'ppt', 'rtf'];
```

Comment:
should this also be extended to other malicious file extensions like .exe etc?

Reply:
Thanks @larowlan.